### DISCUSSION TITLE
Minio question.

### DISCUSSION THREAD

**Discussion by Comandante1977** [outline/outline#3680] [Self Hosting]
Hi all!

Dont see avatar.
![image](https://user-images.githubusercontent.com/45628795/174865657-b66b9567-7cb2-4150-b9a0-9639c935cb9b.png)
It uploading normally and I see it in Minio bucket folder, but not in Outline.
As I read in https://github.com/outline/outline/discussions/2447 it may be nginx configuration is not correct.
Can anybody show me working nginx config for minio?

Minio is staring with these options:
MINIO_OPTS="-C /etc/minio --address ip address here:9000 --console-address ":9010""
Listning on external interface.

Outline config:
AWS_ACCESS_KEY_ID=username
AWS_SECRET_ACCESS_KEY=password
AWS_REGION=eu-central-1
AWS_S3_ACCELERATE_URL=
AWS_S3_UPLOAD_BUCKET_URL=https://address here which
AWS_S3_UPLOAD_BUCKET_NAME=wikibuck
AWS_S3_UPLOAD_MAX_SIZE=26214400
AWS_S3_FORCE_PATH_STYLE=true
AWS_S3_ACL=private

**Comment by almereyda** [comment]
Would

    mc policy set download minio/outline

help to circumvent this error?

This idea is taken from

- https://github.com/outline/outline/discussions/3281#discussioncomment-2430516

where we also find a little more context for setting up Minio correctly.

**Reply by Comandante1977** [reply]
Rather good solution. But, I've solved problem simply by changing Access Policy of the bucket in Minio console from private to public.
![image](https://user-images.githubusercontent.com/45628795/177428862-3232d719-98c4-4901-a6da-1420d83e3dc1.png)

In any way, reading Minio manual (as I did also) is very useful.

Thanks a lot. Your answer turn me in right way.
